### PR TITLE
Split Jobs to folders

### PR DESCRIPTION
**Pull request by radepal**

### CODE REVIEW

File: jenkins/jenkins.yaml.tpl [hunk 1]
```diff
@@ -146,14 +152,52 @@ jobs:
...
-                branch("0.3.0")
+                branch("0.4.0")
               }
             }
             scriptPath("application/kubernetes/install_eks/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Generate_JX_Docker_Image") {
+      pipelineJob("Extras/Kubernetes_Install") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Please rename to Kubernetes_Install_kops.

File: jenkins/jenkins.yaml.tpl [hunk 2]
```diff
@@ -146,14 +152,52 @@ jobs:
...
+                  credentials("bitbucket-key")
+                }
+                branch("0.4.0")
+              }
+            }
+            scriptPath("operations/kubernetes/install_kops/Jenkinsfile")
+          }
+        }
+       }
+      pipelineJob("Extras/Kubernetes_Destroy") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Please rename to Kubernetes_Destroy_kops.

File: jenkins/jenkins.yaml.tpl [hunk 3]
```diff
@@ -89,14 +95,14 @@ jobs:
...
-                branch("0.3.0")
+                branch("0.4.0")
               }
             }
             scriptPath("application/iam/create/${iam_jobs_path}/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Kubernetes_Install") {
+      pipelineJob("EKS/Kubernetes_Install") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Please put EKS Kubernetes jobs under Infrastructure (as K8s cluster is part of infra).

File: jenkins/jenkins.yaml.tpl [hunk 4]
```diff
@@ -210,7 +254,7 @@ jobs:
           }
         }
        }
-      pipelineJob("Grafana_Install") {
+      pipelineJob("Extras/Grafana_Install") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Please create "LMA" folder for Grafana and Prometheus.
Also rename "Install" to "Deploy".

File: jenkins/jenkins.yaml.tpl [hunk 5]
```diff
@@ -58,7 +58,13 @@ credentials:
               privateKey: $${GITPRIVATEKEY}
 jobs:
   - script: >
-      pipelineJob("Generate_IAM_Policies_Operations") {
+      folder('Infrastructure')
+      folder('LMA')
+      folder('Deployment')
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: not needed.

File: jenkins/jenkins.yaml.tpl [hunk 6]
```diff
@@ -89,14 +95,14 @@ jobs:
...
-                branch("0.3.0")
+                branch("0.4.0")
               }
             }
             scriptPath("application/iam/create/${iam_jobs_path}/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Kubernetes_Install") {
+      pipelineJob("Infrastructure/Kubernetes_Install_EKS_Operation_Account") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Operations

File: jenkins/jenkins.yaml.tpl [hunk 7]
```diff
@@ -108,14 +114,14 @@ jobs:
...
+                branch("0.4.0")
               }
             }
-            scriptPath("operations/kubernetes/install/Jenkinsfile")
+            scriptPath("operations/kubernetes/install_eks/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Kubernetes_Destroy") {
+      pipelineJob("Infrastructure/Kubernetes_Destroy_EKS_Operation_Account") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Operations

File: jenkins/jenkins.yaml.tpl [hunk 8]
```diff
@@ -146,14 +152,52 @@ jobs:
...
-                branch("0.3.0")
+                branch("0.4.0")
               }
             }
             scriptPath("application/kubernetes/install_eks/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Generate_JX_Docker_Image") {
+      pipelineJob("Experimental/Kubernetes_Install_KOPS_Operation_Account") {
```
Comment (mzawadzk) [inline jenkins/jenkins.yaml.tpl]: Operations

File: jenkins/jenkins.yaml.tpl [hunk 9]
```diff
@@ -146,14 +160,115 @@ jobs:
...
+                branch("0.4.0")
               }
             }
             scriptPath("application/kubernetes/install_eks/Jenkinsfile")
           }
         }
        }
-      pipelineJob("Generate_JX_Docker_Image") {
+      pipelineJob("Infrastructure/Destroy_Kubernetes_EKS_Application_Account") {
+        displayName('Create EKS cluster in Application Account')
```
Comment (mietlinski-nc) [inline jenkins/jenkins.yaml.tpl]:
```suggestion
        displayName('Remove EKS cluster in Application Account')
```

**Pull request merged**